lib/fun_with_flags/store/persistent.ex

```diff
@@ -1,7 +1,5 @@
 defmodule FunWithFlags.Store.Persistent do
   @moduledoc false
 
-  @adapter FunWithFlags.Config.persistence_adapter
-
-  def adapter, do: @adapter
+  def adapter, do: FunWithFlags.Config.persistence_adapter
```

Comment:
The `@adapter` attribute was placed here just because it was the right scope, semantically speaking. Since we're making this dynamic, we can avoid a chain of function calls by removing this and having the callers use `Config.persistence_adapter` directly.

Reply:
Cool